pmm-app/tests/verifyPMMDemo_test.js

```diff
@@ -0,0 +1,49 @@
+Feature('Verify PMMDemo side');
+
+Scenario(
+  'PMM-T363 - Verify Copyrights & Legal section elements @not-pr-pipeline @pmm-demo',
+  async (I, pmmDemoPage) => {
+    I.amOnPage(pmmDemoPage.url);
+    I.waitForVisible(pmmDemoPage.fields.title, 30);
+    pmmDemoPage.verifyCopyrightsAndLegal();
+    I.amOnPage(pmmDemoPage.url + pmmDemoPage.mongoDBDashbordUrl);
+    pmmDemoPage.verifyCopyrightsAndLegal();
+  }
+);
+
+Scenario(
+  'PMM-T364 - Verify PMM settings returns Access denied error @not-pr-pipeline @pmm-demo',
+  async (I, pmmDemoPage, pmmSettingsPage) => {
+    I.amOnPage(pmmDemoPage.url + pmmSettingsPage.url);
+    I.waitForVisible(pmmDemoPage.fields.accessDenied, 30);
+  }
+);
+
+Scenario(
+  'PMM-T364 - Verify PMM settings returns Access denied error @not-pr-pipeline @pmm-demo',
+  async (I, pmmDemoPage, pmmInventoryPage) => {
+    I.amOnPage(pmmDemoPage.url + pmmInventoryPage.url);
+    I.waitForVisible(pmmDemoPage.fields.accessDenied, 30);
+  }
+);
```

Comment:
```suggestion
```

Reply:
I must be a duplicate of the scenario right above

Reply:
Sorry, I forgot to change test case number. It's not duplicate that's two similar tests. Will push change right now. :)